pennylane/drawer/utils.py

```diff
@@ -44,8 +46,14 @@ def default_bit_map(tape):
     Returns:
         dict: map from mid-circuit measurements to classical wires."""
     bit_map = {}
+    mcms = {}
```

Comment:
The changes in this function are unrelated to the main changes. They just avoid us looping over the entire tape a second time, and instead iterating over the smaller `bit_map` in the second loop (dict comprehension now).

Reply:
Haha, once I got it I was definitely surprised how long I took to get there :sweat_smile: